cabby/geo/item.py

```diff
@@ -37,6 +37,7 @@ class RVSPath:
   main_pivot: Dict = attr.ib()
   near_pivot: Dict = attr.ib()
   beyond_pivot: Dict = attr.ib()
+  cardinal_direction: Text = attr.ib()
```

Comment:
Need to add this to the class documentation. Please include some indication of what possible values are -- e.g. is it just north, south, east, and west, or does it include south-west, etc?

Reply:
Added cardinal description and possible values.

Reply:
Added cardinal description and possible values.